book/src/user/lookup-tables.md

```diff
@@ -0,0 +1,7 @@
+# Lookup tables
+
+In normal programs, you can trade memory for CPU to improve performance, by pre-computing
+and storing lookup tables for some part of the computation. We can do the same thing in
+halo2 circuits!
+
+TODO
```

Comment:
I want to emphasize in this section that it is better to think of a lookup table as representing a relation, rather than a function.